src/pages/NewBeer.jsx

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios'
+import React, { useState } from 'react'
+import Header from '../components/Header'
+
+function NewBeer(props) {
+  const [name, setName] = useState('')
+  const [tagline, setTagline] = useState('')
+  const [description, setDescription] = useState('')
+  const [firstBrewed, setFirstBrewed] = useState('')
+  const [brewersTips, setBrewersTips] = useState('')
+  const [attenuationLevel, setAttenuationLevel] = useState('')
+  const [contributedBy, setContributedBy] = useState('')
+  const [imageUrl, setImageUrl] = useState('')
+  const [success, setSuccess] = useState(false)
+
+  function handleSubmit(event) {
+    event.preventDefault()
+    try {
+      axios.post('https://ih-beers-api2.herokuapp.com/beers/new', {name, tagline, description, first_brewed: firstBrewed, brewers_tips:brewersTips, attenuation_level:attenuationLevel, image_url: imageUrl, contributed_by: contributedBy})
+      .then(result => {
+        setSuccess(true)
+      })
+    }
+    catch (err) {
+      console.log(err)
+    }
+  }
+  return (
+    <div>
+      <Header></Header>
+      <div className='newBeer'>
+        <form onSubmit={handleSubmit} method='POST'>
+          <p>Name</p>
```

Comment:
But I would also wrap the `<input>` with the `<label>` so it creates a better readability for screen readers for example, and also with you click the name that will place the focus on the input 

```suggestion
          <label>Name</label>
```